### PR TITLE
Model submission for R3 sensor configuration 1 from Explorer

### PR DESCRIPTION
**Pull request by HongbiaoZ**
Model submission for R3 sensor configuration 1 by Explorer

### CODE REVIEW

File: submitted_models/explorer_r3_sensor_config_1/model.sdf [hunk 1]
```diff
@@ -0,0 +1,891 @@
...
+      <velocity_decay/>
+      <velocity_decay/>
+      <sensor name="front_laser" type="gpu_lidar">
+              <pose>0.310583 0.001835 0.44788 5e-06 0 -0</pose>
+              <always_on>1</always_on>
+              <update_rate>10</update_rate>
+              <ray>
+                <scan>
+                  <horizontal>
+                    <samples>10000</samples>
```
Comment (AravindaDP) [inline submitted_models/explorer_r3_sensor_config_1/model.sdf:205]: This should be 1800 at 10Hz (If based on VLP-16)

http://www.mapix.com/wp-content/uploads/2018/07/63-9229_Rev-H_Puck-_Datasheet_Web-1.pdf

Comment (HongbiaoZ) [reply submitted_models/explorer_r3_sensor_config_1/model.sdf:205]: Yes, it's a mistake. It should be 1800.

Comment (AravindaDP) [reply submitted_models/explorer_r3_sensor_config_1/model.sdf:205]: While we are at it better to update DS1, R2 and X1 as well (1200 at 15Hz) (may be in a separate PR)

https://github.com/osrf/subt/blob/master/submitted_models/explorer_ds1_sensor_config_1/model.sdf#L585
https://github.com/osrf/subt/blob/master/submitted_models/explorer_r2_sensor_config_1/model.sdf#L197
https://github.com/osrf/subt/blob/master/submitted_models/explorer_x1_sensor_config_1/model.sdf#L580
https://github.com/osrf/subt/blob/master/submitted_models/explorer_x1_sensor_config_2/model.sdf#L580

**Comment by acschang** [comment]
Merging to another PR since the originating branch of this PR is inaccessible.

**Comment by acschang** [comment]
Check the new PR #734

**Pull request merged**